core/spec/models/spree/store_credit_spec.rb

```diff
@@ -897,4 +897,11 @@
       end
     end
   end
+
+  describe "#generate_authorization_code" do
+    it "doesn't rely on time for uniqueness" do
+      expect(Time).to receive(:current).twice { Time.zone.parse("2019/09/05 06:30:00") }
```

Comment:
Should we use the Rails helpers for stubbing time here, so the test is not so closely coupled with the implementation?

Reply:
I actually wanted to explicitly couple it with the implementation, because the problem is with it. The moment we stop using the timestamp, this specific spec implementation may not make much sense anymore (I mean the line after this one), surely it won't make sense its text description.

Reply:
What I meant is that there's a ton of different ways to get the current time in Ruby/Rails: `Time.current`, `Time.now`, `Time.zone.now`, `DateTime.current`, etc. In the test, we shouldn't worry about what method is being called exactly in the implementation.

A quick way to solve the problem would be to use the [helpers](https://api.rubyonrails.org/v5.2.5/classes/ActiveSupport/Testing/TimeHelpers.html) Rails provides for stubbing time.

Does that make sense?

Reply:
Oh, sure! I think I knew about those helpers at a certain point in my life, but I forgot about them since then.